mir_eval/multipitch.py

```diff
@@ -0,0 +1,383 @@
+'''
+The goal of multiple f0 (multipitch) estimation and tracking is to identify
+all of the active fundamental frequencies in each time frame in a complex music
+signal.
+
+Conventions
+-----------
+Multipitch estimates are represented by a time series, and a corresponding list
+of arrays of frequency estimates. Frequency estimates may have any number of
+frequency values, including (represented by an empty array). Time values are in
+units of seconds and frequency estimates are in units of Hz.
+
+Estimate time series should ideally be equal to reference time series, but if
+this is not the case, the estimate time series is resampled using a nearest
+neighbor interpolation to match the estimate. Time values in the estimate time
+series that are outside of the range of the reference time series are given
+null (empty array) frequencies.
+
+By default, a frequency is "correct" if it is within 0.5 semitones of a
+reference frequency. The metrics are based on those described in
+[#poliner2007]_ and [#bay2009]_.
+
+Metrics
+-------
+* :func:`mir_eval.multipitch.accuracy`: Precision, Recall, and Accuracy scores
```

Comment:
Huh, interesting.  I am going to assume that some smart person who knows about multipitch eval made that decision a while ago and am ok with sticking with it.